### PR TITLE
Marcelle/synapse mci conversion

### PR DESCRIPTION
**Pull request by m-goggins**
Reads a parquet file containing incident information and appends it to the `mci_delta_table_path`. 

Currently, we would need to manually change the `mci_incoming_file_path` whenever LAC uploads a file. Additionally, the `mci_delta_table_path` needs to be updated once that delta table is created.

Also snuck in changes to the MPI notebook so that reading parquet files methods, e.g., `spark.read...` is consistent for both the MPI and MCI.

### CODE REVIEW

File: .github/workflows/deployment.yaml [hunk 1]
```diff
@@ -193,6 +193,8 @@ jobs:
         run: |
           if ! az synapse notebook show --workspace-name phdi${TF_ENV}synapse${SHORT_CID} --name "Convert Parquet MPI"; then
             az synapse notebook create --workspace-name phdi${TF_ENV}synapse${SHORT_CID} --name "Convert Parquet MPI" --file "scripts/Synapse/convertParquetMPI.ipynb"
+          if ! az synapse notebook show --workspace-name phdi${TF_ENV}synapse${SHORT_CID} --name "Convert Parquet MCI"; then
```
Comment (nickclyde) [inline .github/workflows/deployment.yaml:196]: Oops, looks like this is missing a `fi` to close out the previous `if` statement before starting the next one!

**Review by nickclyde — CHANGES_REQUESTED** [review]
Awesome work Marcelle! Just one small change.

File: .github/workflows/deployment.yaml (hunk 1 above)
Comment (m-goggins) [reply .github/workflows/deployment.yaml:197]: Good catch! Just added

**Review by nickclyde — APPROVED** [review]
:shipit:

**Pull request merged**